vendor_skins/Evercast/CI/before-script-osx.sh

```diff
@@ -3,14 +3,16 @@ export PATH=/usr/local/opt/ccache/libexec:$PATH
 mkdir build
 cd build
 cmake \
--DDepsPath=/tmp/obsdeps \
+-DENABLE_SCRIPTING=OFF \
+-DDepsPath=~/obsdeps \
 -DCMAKE_BUILD_TYPE=RELEASE \
 -DCMAKE_INSTALL_PREFIX=/opt/ebs \
--DVLCPath=$PWD/../../vlc-master \
--DQTDIR=/usr/local/Cellar/qt/5.10.1 \
--DCMAKE_OSX_DEPLOYMENT_TARGET=10.10 \
+-DVLCPath=~/vlc-3.0.4 \
+-DQTDIR=/usr/local/Cellar/qt/5.14.1 \
```

Comment:
Is this the OBS Upstream Qt version?  I think there's something in cmake that enables usage of the latest Qt especially integration with Qt Designer or whatever their UI is.  Just curious / sharing info

Reply:
Yes, this is what they are using in later versions.